cmd/dlv/main.go

```diff
@@ -49,5 +48,92 @@ func main() {
 		os.Exit(0)
 	}
 
-	cli.Run(os.Args[1:])
+	// Create process launch args from the CLI args
+	launchArgs, err := buildLaunchArgs(flag.Args())
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	// Start the debugger
+	stopDebugger := make(chan bool)
```

Comment:
This (59-68) should be moved into the service that consumes the debugger, not here. The service should be responsible for this type of setup/cleanup.